dm/pkg/binlog/position.go

```diff
@@ -274,6 +274,13 @@ func (l Location) Clone() Location {
 	return l.CloneWithFlavor("")
 }
 
+// Clone clones a same location, but reset suffix to 0.
```

Comment:
please left a todo location whit suffix is hadr do understand, let's refine this logic later

Reply:
yes, suffix logic is hard to understand and it is used in compare location too much. i think i will refine it in other pr.